package.json

```diff
@@ -27,7 +27,7 @@
     "launchdarkly-js-client-sdk": "^2.18.3",
     "lodash": "^4.17.20",
     "remove-accents": "^0.4.2",
-    "sbc-common-components": "2.4.17",
```

Comment:
I am OK with not updating the app version, since the Sbc Common Components version also displays in the footer "i" tooltip (so we can verify when it's deployed).

Reply:
I would ask about it :)

Reply:
We implemented that feature for internal use. It still does what it needs to do. No need to ask anyone else :)